coordinator/src/services/communication/live-spine-client.ts

```diff
@@ -11,14 +11,7 @@ const SPINE_ENDPOINT = process.env.SPINE_URL
 const SPINE_PATH = "Prescription"
 const BASE_PATH = process.env.BASE_PATH
 
-const httpsAgent = new https.Agent({
-  cert: process.env.CLIENT_CERT,
-  key: process.env.CLIENT_KEY,
-  ca: [
-    process.env.ROOT_CA_CERT,
-    process.env.SUB_CA_CERT
-  ]
-})
+const httpsAgent = new https.Agent()
```

Comment:
Presumably don't even need to pass in an agent anymore, we can just use the default one

Reply:
Done